.yamato/hdrp-osx-metal.yml

```diff
@@ -46,82 +46,6 @@ HDRP_OSX_Metal_playmode_mono_Linear_trunk:
     dependencies:
       -  path: .yamato/_editor_priming.yml#editor:priming:trunk:OSX
          rerun: on_new_revision
-HDRP_OSX_Metal_playmode_XR_mono_Linear_trunk:
```

Comment:
Was the removal of xr tests on mac intended?

Reply:
Yes, they're not monitored or supported for now.